cylc/flow/etc/job.sh

```diff
@@ -187,6 +205,20 @@ cylc__set_return() {
     return "${1:-0}"
 }
 
+###############################################################################
+# Save the data using cylc message and exit the profiler
+cylc__kill_profiler() {
+    if [[ -n "${cpu_time:-}" ]]; then
+      cylc message -- "${CYLC_WORKFLOW_ID}" "${CYLC_TASK_JOB}" "DEBUG: cpu_time $cpu_time" || true
+    fi
+    if [[ -n "${max_rss:-}" ]]; then
+      cylc message -- "${CYLC_WORKFLOW_ID}" "${CYLC_TASK_JOB}" "DEBUG: max_rss $max_rss" || true
```

Comment:
It would be advantageous if we could send one message rather than two (lower per-job overhead).

A simple way to do this would be to get the profiler to write its results into a single file in a format like JSON (e.g `{"memory": x, "cpu": y}`, then dump the contents of that file into the `cylc message` command.

Reply:
Done